pkg/kubelet/kuberuntime/kuberuntime_sandbox_test.go

```diff
@@ -85,7 +85,6 @@ func TestCreatePodSandbox_RuntimeClass(t *testing.T) {
 			id, _, err := m.createPodSandbox(pod, 1)
 			if test.expectError {
 				assert.Error(t, err)
-				assert.Contains(t, fakeRuntime.Called, "RunPodSandbox")
```

Comment:
Can you also update the test so this would fail? I think you can just move the initialization of the fake runtime into this test run